src/main/java/teammates/common/util/Const.java

```diff
@@ -1371,30 +1027,15 @@ private Const() {
         public static final String STUDENT_COURSE_JOIN_SUCCESSFUL = "You have been successfully added to the course %s.";
 
         /** Template String. Parameters:  Course ID */
-        public static final String NON_EXISTENT_STUDENT_ATTEMPTING_TO_JOIN_COURSE =
-                "Unable to join course %s as you are currently not in the student list of that course. "
-                + "Please contact your course instructor for assistance.";
         public static final String STUDENT_PROFILE_NOT_A_PICTURE = "The file that you have uploaded is not a picture. "
                 + "Please upload a picture (usually it ends with .jpg or .png)";
         public static final String STUDENT_PROFILE_NO_PICTURE_GIVEN = "Please specify a file to be uploaded.";
         public static final String STUDENT_NOT_FOUND_FOR_RECORDS =
                 "The student you tried to view records for does not exist. " + STUDENT_EVENTUAL_CONSISTENCY;
-        public static final String STUDENT_PROFILE_PICTURE_EDIT_FAILED =
-                "The photo that was edited did not belong to the user. "
-                + "Please upload another picture to begin editing";
 
         public static final String UNREGISTERED_STUDENT_MESSAGE = "You may submit feedback for sessions "
```

Comment:
Once you remove UNREGISTERED_STUDENT and UNREGISTERED_STUDENT_RESULTS, this variable becomes unused too.